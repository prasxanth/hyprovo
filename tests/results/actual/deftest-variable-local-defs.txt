2021-11-24 13:52:30.117 - INFO - Pass ✓... (test-sub-variable-local-defs): sub pass
2021-11-24 13:52:30.119 - ERROR - Fail ✗... (test-sub-variable-local-defs): sub fail 
                                    Expected: 7
                                    Got: -2
