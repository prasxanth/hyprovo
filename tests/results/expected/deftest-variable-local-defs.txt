2021-11-24 12:39:44.272 - INFO - Pass ✓... (test-sub-variable-local-defs): sub pass
2021-11-24 12:39:44.275 - ERROR - Fail ✗... (test-sub-variable-local-defs): sub fail 
                                    Expected: 7
                                    Got: -2
